models/daily_spend.sql

```diff
@@ -245,7 +282,8 @@ search_optimization_spend_daily as (
         null as storage_type,
         null as warehouse_name,
         null as database_name,
-        coalesce(sum(stg_metering_history.credits_used * daily_rates.effective_rate), 0) as spend
+        coalesce(sum(stg_metering_history.credits_used * daily_rates.effective_rate), 0) as spend,
+        spend as spend_net_cloud_services
     from dates
     left join {{ ref('stg_metering_history') }} on
         dates.date = convert_timezone('UTC', stg_metering_history.start_time)::date
```

Comment:
@NiallRees some i not able to test run right now due to:

```
❯ dbt run -m daily_spend
02:16:06  Running with dbt=1.3.0
02:16:06  Found 21 models, 2 tests, 0 snapshots, 0 analyses, 424 macros, 0 operations, 0 seed files, 11 sources, 0 exposures, 0 metrics
02:16:06  
02:16:07  Concurrency: 8 threads (target='dev')
02:16:07  
02:16:07  1 of 1 START sql table model public.daily_spend ................................ [RUN]
02:16:08  Unhandled error while executing model.dbt_snowflake_monitoring.daily_spend
maximum recursion depth exceeded
02:16:08  1 of 1 ERROR creating sql table model public.daily_spend ....................... [ERROR in 1.03s]
02:16:08  
02:16:08  Finished running 1 table model in 0 hours 0 minutes and 2.35 seconds (2.35s).
02:16:08  
02:16:08  Completed with 1 error and 0 warnings:
02:16:08  
02:16:08  maximum recursion depth exceeded
02:16:08  
02:16:08  Done. PASS=0 WARN=0 ERROR=1 SKIP=0 TOTAL=1
```

googling suggests this has something to do with python 3.10

were you able to run this successfully? what versions are you using? here's mine:

```
❯ dbt --version
Core:
  - installed: 1.3.0
  - latest:    1.4.1 - Update available!

  Your version of dbt-core is out of date!
  You can find instructions for upgrading here:
  https://docs.getdbt.com/docs/installation

Plugins:
  - snowflake: 1.3.0 - Update available!

  At least one plugin is out of date or incompatible with dbt-core.
  You can find instructions for upgrading here:
  https://docs.getdbt.com/docs/installation
```

Reply:
Have you run dbt deps?